### PR TITLE
Update global-config.js

### PR DESCRIPTION
**Pull request by SerkanSipahi**
show warn if jspm config is used and no <options>, <setting> were passed

### CODE REVIEW

**Comment by SerkanSipahi** [comment]
@guybedford do you know why is appveyor is failing?

File: lib/global-config.js [hunk 1]
```diff
@@ -125,6 +125,11 @@ function save() {
 exports.save = save;
 
 exports.set = function(name, val) {
+ 
+  if(!(name && val)) {
+     console.warn('Please pass option and setting value! See for more information jspm --help');
+  }
```
Comment (guybedford) [inline lib/global-config.js]: This check should be in https://github.com/jspm/jspm-cli/blob/0.17/cli.js#L590 I think rather, where we could perhaps include something like:

```
ui.warn('jspm config requires a property and value via %jspm config property value%')
```

or similar.

**Comment by guybedford** [comment]
Thanks for the PR, this seems useful!

The Appveyor build is a little unstable at times unfortunately.

**Comment by SerkanSipahi** [comment]
@guybedford 
>The Appveyor build is a little unstable at times unfortunately.
I would be better if we remove "Appveyor" for a while as long as "Appveyor" is not stable!

**Pull request closed without merging**